retworkx-core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "retworkx-core"
-version = "0.11.0"
+version = "0.12.0"
```

Comment:
It does seem weird to backport a 0.12 version bump to the 0.11 branch, perhaps we should use something like 0.11.1?

Reply:
Yeah, we have to manually fix the version strings here. I wanted to backport the version in the cargo.toml and not the bump on retworkx-core